plugins/inputs/consul/consul.go

```diff
@@ -18,19 +18,19 @@ import (
 var sampleConfig string
 
 type Consul struct {
-	Address    string
-	Scheme     string
-	Token      string
-	Username   string
-	Password   string
-	Datacentre string `toml:"datacentre" deprecated:"1.10.0;1.35.0;use 'datacenter' instead"`
-	Datacenter string
-	tls.ClientConfig
-	TagDelimiter  string
-	MetricVersion int
+	Address       string `toml:"address"`
+	Scheme        string `toml:"scheme"`
+	Token         string `toml:"token"`
+	Username      string `toml:"username"`
+	Password      string `toml:"password"`
+	Datacentre    string `toml:"datacentre" deprecated:"1.10.0;1.35.0;use 'datacenter' instead"`
+	Datacenter    string `toml:"datacenter"`
+	TagDelimiter  string `toml:"tag_delimiter"`
+	MetricVersion int    `toml:"metric_version"`
```

Comment:
Do all of these need to gain toml metadata?

Reply:
It seems that this is the convention in structs that 'configure' plugins.